vaultwarden/templates/ingress.yaml

```diff
@@ -16,6 +18,9 @@ metadata:
   {{- end }}
 spec:
   {{- if .Values.ingress.tls }}
+  {{- if .Values.ingress.className }}
```

Comment:
Currently this will only add "className" if TLS options are present. Is this the intended behavior? Otherwise move the block outside of the TLS condition.